kazel/generator.go

```diff
@@ -66,9 +66,20 @@ func (v *Vendorer) findOpenAPI(root string) ([]string, error) {
 	}
 	var res []string
 	var includeMe bool
+	var skip bool
 	for _, finfo := range finfos {
 		path := filepath.Join(root, finfo.Name())
+		skip = false
 		if finfo.IsDir() && (finfo.Mode()&os.ModeSymlink == 0) {
+                        for _, r := range v.skippedPaths {
```

Comment:
there's already a check for this on L58-62 - is that not sufficient?

Reply:
Nope, it checks only for the root path, we want to do this for all dirs traversed. I should probably remove that check.

Reply:
we call `findOpenAPI` recursively, though?

Reply:
Ah, okay. I get it. This is because it needs to check for just the dir name instead of whole path. The `root` argument has the whole path.

Reply:
can you use something like `/_examples` in `SkippedPaths`?

Reply:
I debugged this, the value of `root` for this case is `.make/vendor/k8s.io/code-generator/_examples`. Kubernetes currently uses for `^_.*` SkippedPaths - https://github.com/kubernetes/kubernetes/blob/master/build/root/.kazelcfg.json#L3
Instead of adding all the specific paths that start with `_` in`SkippedPaths`, best would be to just check for dir name.

Reply:
I've refactored the code. PTAL.

Reply:
Even for the built-in skips we assume that the check will be on the base path - https://github.com/kubernetes/repo-infra/blob/2a736b4fba317cf3038e3cbd06899b544b875fae/kazel/kazel.go#L134